docs/src/modules/components/ApiPage.tsx

```diff
@@ -169,19 +167,9 @@ export default function ApiPage(props: ApiPageProps) {
     createTocEntry('demos'),
     createTocEntry('import'),
     ...componentDescriptionToc,
-    getPropsToC({
-      t,
-      componentName: pageContent.name,
-      componentProps,
-      inheritance,
-      themeDefaultProps: pageContent.themeDefaultProps,
-    }),
+    createTocEntry('props'),
     componentSlots?.length > 0 && createTocEntry('slots'),
-    ...getClassesToC({
-      t,
-      componentName: pageContent.name,
-      componentClasses,
-    }),
+    createTocEntry('classes'),
```

Comment:
Need to bring that feature back. I initially removed it because I remembered that with the table layout it was useless, but did not manage to find back this conclusion on github